word-search.py

```diff
@@ -237,4 +237,40 @@ def word_search():
     valid_words = load_word_library(grid)
   
 
+def continue_or_restart(): #function to enable the player to continue or quit in-between guesses
+    while True:
+        option = input("\nContinue Game? (y/n): ").strip().lower()
+        if option == 'y':
+            return True  #continue playing
+        elif option == 'n':
+            return False  #quit
+        else:
+            print("Invalid input. Please enter 'y' to continue or 'n' to restart.")
+
+def new_game():
+    while True: 
+        gridsize, timeroption = menu() #get grid size and timer option
+        word_search()  # start game
```

Comment:
Menu is also called inside the word search, hence when played user is prompted twice for the grid size and timer. Maybe the flow can be unified :))

Reply:
yes, para hindi mag D.R.Y., actually na notice ko yan ginawa ko lang uli para di ma galaw yung existing codes.

Reply:
yung word_search() i-omit na ilagay na sa new_game()

Reply:
I think it is okay to update the existing code specifically the word_search  para po hindi doble yung menu?  or like u said po omit word search and include it sa new_game :)

Reply:
yes, let's also wait for vickie's call as well, since kaya nga may branches.

Reply:
noted on this, will get it done within the day, thank you both.